ui/app/components/app/transaction-activity-log/transaction-activity-log.component.js

```diff
@@ -71,20 +71,13 @@ export default class TransactionActivityLog extends PureComponent {
   renderActivity (activity, index) {
     const { conversionRate, nativeCurrency } = this.props
     const { eventKey, value, timestamp, hash } = activity
-    const ethValue = index === 0
-      ? `${getValueFromWeiHex({
-        value,
-        fromCurrency: nativeCurrency,
-        toCurrency: nativeCurrency,
-        conversionRate,
-        numberOfDecimals: 6,
-      })} ${nativeCurrency}`
-      : getEthConversionFromWeiHex({
```

Comment:
Why remove this condition? Now all of the activity log entries are shown in GWEI instead of ETH.

I don't really understand the decision to show ETH for the first one and GWEI for the rest, but best to keep design changes separate from the bug fix either way. Swapping out `nativeCurrency` for `ETH` in both of these `getValueFromWeiHex` calls seems to fix the problem.